`pub use bio-types` wherever they are used to avoid dependency hell

As part of nextstrain/nextclade we use bio's gff3 parser.

That parser uses the Strand type from bio-types in its public API.

However, that Strand type is not `pub use`d but just `use`d which can cause dependency hell.

For example, it happened that when I `cargo update`d the Nextclade repo, the bio-types dependency for bio was bumped to `0.13.0` while our `Cargo.toml` pins `bio-types` to `0.12.1`. So now we end up with a compiler error because the versions differ:

```
error[E0631]: type mismatch in function arguments
   --> packages_rs/nextclade/src/io/gff3.rs:34:57
    |
34  |     strand: record.strand().map_or(GeneStrand::Unknown, Strand::into),
    |                             ------                      ^^^^^^^^^^^^
    |                             |                           |
    |                             |                           expected due to this
    |                             |                           found signature defined here
    |                             required by a bound introduced by this call
    |
    = note: expected function signature `fn(bio_types::strand::Strand) -> _`
               found function signature `fn(Strand) -> _`
note: required by a bound in `std::option::Option::<T>::map_or`
   --> /Users/corneliusromer/.rustup/toolchains/1.66.0-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:983:12
    |
983 |         F: ~const FnOnce(T) -> U,
    |            ^^^^^^^^^^^^^^^^^^^^^ required by this bound in `std::option::Option::<T>::map_or`

For more information about this error, try `rustc --explain E0631`.
error: could not compile `nextclade` due to previous error
```

Stackoverflow says the solution would be for `bio` to `pub use bio-types` - then we could avoid the direct dependency on `bio-types` and make sure to use the same one.

I think you do this in some places but not in others. Would be good to be consistent and `pub use` wherever `bio-types` are exposed in the public API.

Here you should `pub use`: https://github.com/rust-bio/rust-bio/blob/bd08234ca777c0dad8db9b0a5a16e2e4a0e9eefc/src/io/gff.rs#L37

Here you do it correctly I think: https://github.com/rust-bio/rust-bio/blob/bd08234ca777c0dad8db9b0a5a16e2e4a0e9eefc/src/alignment/mod.rs#L14

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -261,3 +261,4 @@ pub mod scores;
 pub mod seq_analysis;
 pub mod stats;
 pub mod utils;
+pub use bio_types;

